bears/python/PycodestyleBear.py

```diff
@@ -0,0 +1,48 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pycodestyle',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) (?P<column>\d+) (?P<message>.*)')
+class PycodestyleBear:
+    """
+    A wrapper for the tool ``pycodestyle`` formerly known as ``pep8``.
+    """
+    LANGUAGES = {"Python", "Python 2", "Python 3"}
+    REQUIREMENTS = {PipRequirement('pycodestyle')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    @staticmethod
+    def create_arguments(
+            filename, file, config_file,
+            pycodestyle_ignore: str="",
+            pycodestyle_select: str="",
+            max_line_length: int=None):
+        """
+        :param pycodestyle_ignore:
+            Comma separated list of errors to ignore.
+            See ``pydocstyle`` documentation for a complete list of errors.
+        :param pycodestyle_select:
+            Comma separated list of errors to detect. If given only
+            these errors are going to be detected.
+            See ``pydocstyle`` documentation for a complete list of errors.
+        """
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+
+        if pycodestyle_ignore:
+            arguments.append("--ignore=" + pycodestyle_ignore)
+
+        if pycodestyle_select:
+            arguments.append("--select=" + pycodestyle_select)
+
+        if max_line_length:
```

Comment:
I'd remove this if and give it a proper default on top, this way people can also see in the doc what's done by default and if they want to modify it